c2corg_ui/externs/appx.js

```diff
@@ -84,3 +84,186 @@ appx.BootstrapSliderValues;
  * }}
  */
 appx.auth.RequestChangePassword;
+
+
+/**
+ * @typedef {{
+ *   routes : Array<Object>,
+ *   waypoints : Array<Object>,
+ *   images : Array<Object>,
+ *   users : Array<Object>,
+ *   recent_outings : Array<Object>,
+ * }}
+ */
+appx.DocumentAssociations;
```

Comment:
those properties you listed do not replace any of the existing properties?
`?` means Something or Null ?